diesel/src/migrations/mod.rs

```diff
@@ -297,6 +297,17 @@ where
     Ok(())
 }
 
+fn format_migration_output(migration: &Migration) -> String {
```

Comment:
I think we should rather implement `Display` for Migration instead of implementing the formating as custom function.

Reply:
Indeed, that's a good suggestion. That would also help us get rid of all the `unwrap` (but we'll have to convert them to `fmt::Error`s which dont carry any information)